Core/include/Acts/Propagator/detail/JacobianEngine.hpp

```diff
@@ -66,22 +49,23 @@ ActsMatrix<8, 7> anglesToDirectionJacobian(const Vector3& direction);
 /// approximated approach to treat the (assumed) small change.
 ///
 /// @param [in] geoContext The geometry Context
+/// @param [in] surface Target surface
 /// @param [in] freeParameters Free, nominal parametrisation
 /// @param [in] boundToFreeJacobian Jacobian from bound to free at start
 /// @param [in] freeTransportJacobian Transport jacobian free to free
 /// @param [in] freeToPathDerivatives Path length derivatives for free
-/// parameters
-/// @param [in] surface Target surface
+///        parameters
+/// @param [out] fullTransportJacobian A 6x6 transport jacobian from bound to bound
```

Comment:
Do you expect this be a performance benefit, instead of returning this matrix?

Reply:
I don't know if this would have any performance implications but I copied this over from the `CovarianceEngine` where the code was duplicated and there we used a reference for returning.